TransIT.BLL/Exceptions/DocumentContentException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TransIT.BLL.Exceptions
+{
+
+    [Serializable]
+    public class DocumentContentException : DocumentException
+    {
+        public DocumentContentException() { }
```

Comment:
fix formatting of code: add enters, spaces

Reply:
fixed formatting in all exceptions